pages/dkp/kommander/2.2/insights/index.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: layout.pug
+navigationTitle: DKP Insights
+title: DKP Insights
+excerpt: Use DKP Insights to monitor and tune your environment
+menuWeight: 55
+beta: false
+enterprise: false
+techPreview: true
+---
+
+<p class="message--note"><strong>NOTE:</strong> We are offering DKP Insights with this release as a Technical preview only. </p>
+
+We designed DKP Insights to assist Kubernetes Administrators with daily tasks, such as ensuring your clusters are secure, debugging, monitoring, and anticipating issues. DKP Insights collects logs, events, and metrics from the DKP Management cluster and managed or attached Kubernetes clusters. It then generates rules-based insights on potential problems of varying critical levels so that you can quickly identify and resolve them yourself.
```

Comment:
This one needs rewording.

Reply:
How about this: "such as ensuring clusters are secure, resources are optimally used, workloads and clusters follow best practices, and resolve any problems that arise"

Reply:
I agree it needs rewording.    But we need to work on the proposed reformulation, because all the phrases need to agree with the verb here, which is 'ensuring'; the phrase 'resolve any problems that arise' doesn't agree.

But I also worry about over-promising here.  Insights can't really 'ensure' anything - it can only surface some issues and the user still has to resolve them themselves.  So I don't think we should say 'ensuring clusters are secure' because this implies to me that insights can 'ensure all possible security issues are surfaced'.

How about this:

```suggestion
We designed DKP Insights to assist Kubernetes Administrators with daily tasks, such as checking for security issues, monitoring resource usage, verifying that workloads and clusters follow best practices, and resolving common issues. DKP Insights collects logs, events, and metrics from the DKP Management cluster and managed or attached Kubernetes clusters. It then generates rules-based insights on potential problems of varying critical levels so that you can quickly identify and resolve them yourself.
```

Reply:
All solid points, thanks @cneth 
I also wonder about the using a persona here: Kube Admins. While in practice this is something good that we want to get into with the docs eventually, we haven't yet used personas in the technical doc and don't want to confuse anyone. 
Is this generally understood and won't throw any users off, or would the ambiguous "you" be better here?
So:
"We designed DKP Insights to assist **you** with..."
It seems minor, but we want to avoid any mental breaks or pauses when a user reads this; ie, don't want them stopping mid-sentence asking "are they talking about me?".

Reply:
@cneth - I forgot to commit this before pulling back into VSC. I have this added and it'll show up in a minute when I push this back up from VSC.